attributes/server_web2.rb

```diff
@@ -11,3 +11,4 @@
 default['icinga2']['web2']['web_uri'] = '/icingaweb2'
 default['icinga2']['web2']['conf_dir'] = '/etc/icingaweb2'
 default['icinga2']['web2']['log_dir'] = '/var/log/icingaweb2'
+default['icinga2']['web2']['release'] = '1'
```

Comment:
This attribute is already evaluated and at the moment only matters to `rhel` family. I am not sure setting it to `1` adds any value to `debian` family.